src/controllers/itemDetails/index.js

```diff
@@ -1842,9 +1842,7 @@ function renderSpecials(page, item, user) {
 }
 
 function renderCast(page, item) {
-    const people = (item.People || []).filter(function (p) {
-        return p.Type === 'Actor';
```

Comment:
Since this PR was originally created, the Cast & Crew and Guest Star lists have been separated on the page.

This means we need to filter out people whose `Type` is 'GuestStar'